doc/Homepage.ipynb

```diff
@@ -52,10 +53,11 @@
    },
    "outputs": [],
    "source": [
+    "%%output backend='matplotlib'\n",
     "%%opts Points [scaling_factor=50] Contours (color='w')\n",
     "dots = np.linspace(-0.45, 0.45, 19)\n",
     "\n",
-    "hv.HoloMap({y: (fractal * fractal.sample([(i,y) for i in dots]).to.points(['x','y'], 'z') +\n",
+    "hv.HoloMap({y: (fractal * hv.Points(fractal.sample([(i,y) for i in dots])) +\n",
```

Comment:
I like this change; I think it's easier to understand, and a bit shorter.  Can you please apply that to master?